schemas/system/gs.domains

```diff
@@ -9,7 +9,7 @@
   Logical: { type: 'boolean', control: 'CheckBox' },
   PostalCode: { type: 'string', length: 10 },
   Money: { type: 'object', class: 'Money' },
-  Nomen: { type: 'string', length: 60 },
+  Nomen: { type: 'string', length: 128 },
```

Comment:
I think it'd be better to create a new domain for purposes where length must be 128. IMO Nomen should be reserved for DB and other identifier related purposes.

Reply:
Agree